src/pages/Admin/AdminLogin/index.jsx

```diff
@@ -1,23 +1,65 @@
 import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom';
 import {
   MDBContainer,
-  MDBTabs,
-  MDBTabsItem,
-  MDBTabsLink,
-  MDBTabsContent,
-  MDBTabsPane,
   MDBBtn,
-  MDBIcon,
   MDBInput,
   MDBCheckbox
 } from 'mdb-react-ui-kit';
 import './index.css';
+import axios from 'axios'; 
 import { Navbar } from "../../../components/admin/navbar";
-
+import { InfinitySpin } from 'react-loader-spinner';
+import toast from 'react-hot-toast';
 
 
 
 const AdminLogin = () => {
+  const [adminLoginData, setAdminLoginData] = useState({
+    email: '',
+    password: '',
+  });
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
+
+  const loginAdmin = async (event) => {
+    try {
+      event.preventDefault();
+      setLoading(false);
```

Comment:
Your loading is meant to be set to true at this point to signify that you're loading something